models/User.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema(
+  {
+    admin: { type: Boolean, default: true },
+    name: {
+      type: String,
+      required: true,
+    },
+    email: {
```

Comment:
Change all these fields to be in one row like you did in the other schemas (just to be consistent)